applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -99,12 +173,6 @@ def SetUpAnalysis(self):
             self.time = 0.0
```

Comment:
Why don't you allow a non-zero starting time?

Reply:
Good point, I should actually read it from JSON (it is there already).